tests/transformers/tests/generation/test_utils.py

```diff
@@ -1942,6 +1947,8 @@ def _check_hidden_states_for_generate(
 
         for idx, iter_hidden_states in enumerate(hidden_states):
             seq_len = min_length + idx if not use_cache else 1
+            if iter_hidden_states[0].shape[-2] == max_length:
```

Comment:
if we wrap this in a self explanatory function it would be more readable. eg : IsPadded()